diff -ruN wavesurfer-1.8.1-old/Makefile wavesurfer-1.8.1/Makefile
--- wavesurfer-1.8.1-old/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ wavesurfer-1.8.1/Makefile	2005-03-16 19:43:12.499058120 +0100
@@ -0,0 +1,9 @@
+
+
+all:
+	
+install:
+	cp wavesurfer.tcl wavesurfer
+	chmod 644 `find wsurf* -name "*.tcl"`
+	chmod 644 wsurf*/configurations/*
+	chmod 644 wsurf*/plugins/*
diff -ruN wavesurfer-1.8.1-old/README.txt wavesurfer-1.8.1/README.txt
--- wavesurfer-1.8.1-old/README.txt	2005-03-15 20:59:06.000000000 +0100
+++ wavesurfer-1.8.1/README.txt	2005-03-16 19:43:12.499058120 +0100
@@ -1,4 +1,5 @@
 WaveSurfer
+==========
 
 Copyright (C) 2000-2005
 Jonas Beskow	beskow@speech.kth.se
